packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -611,6 +612,18 @@ const messages = {
         hasCopiedTitle: 'Embed link copied',
       },
     },
+    gloss: {
+      heading: 'How to reuse glossary content',
+      description: 'Remember that built-in content might have a different license than the glossary text',
```

Comment:
Gloss er eintal og glossary fleirtall
```suggestion
      description: 'Remember that built-in content might have a different license than the gloss text',
```